File: GTTS/INL.PersonService.Models/PersonLanguage_Item.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace INL.PersonService.Models
{
    public class PersonLanguage_Item : IPersonLanguage_Item
    {
        public long PersonID { get; set; }
        public int LanguageID { get; set; }
        public int? LanguageProficiencyID { get; set; }
        public string LanguageCode { get; set; }
        public string LanguageDescription { get; set; }
        public string LanguageProficiencyCode { get; set; }
        public string LanguageProficiencyDescription { get; set; }
        public int ModifiedByAppUserID { get; set; }  
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/person-language_item.ts







import { IPersonLanguage_Item } from './iperson-language_item';

export class PersonLanguage_Item implements IPersonLanguage_Item {
  
	public PersonID: number = 0;
	public LanguageID: number = 0;
	public LanguageProficiencyID?: number;
	public LanguageCode: string = "";
	public LanguageDescription: string = "";
	public LanguageProficiencyCode: string = "";
	public LanguageProficiencyDescription: string = "";
	public ModifiedByAppUserID: number = 0;
  
}

